src/axom/sidre/docs/sphinx/data_vs_metadata.rst

```diff
@@ -44,10 +46,8 @@ as we can see by the output of the code::
 Example 1: One View, One Buffer 
 --------------------------------
 
-The first example illustrates a common Sidre usage pattern in which
-a view is created in a group and the data associated with the view is managed
-via the view. This case represents a one-to-one relationship between a view and
-buffer.
+The first example shows a common Sidre usage pattern in which a view is created in a group and the data associated with the view is managed via the view. This 
+case represents a one-to-one relationship between a view and buffer.
 
 We begin by creating a view "aview" in group "A" describing an integer array 
 of length 10 and allocating the array, all in one method call. This creates a 
```

Comment:
Maybe we should add commas separating the operations.  This emphasizes that we are doing several things at once and makes "all in one method call" more significant.

```suggestion
We begin by creating a view "aview" in group "A", describing an integer array 
of length 10, and allocating the array, all in one method call. This method call creates a 
```

Alternately, we could change the next sentence so it starts either with "This method call creates a buffer..." or with "These operations create a buffer..."

Reply:
changed